addons/sourcemod/scripting/szf/pickupweapons.sp

```diff
@@ -501,6 +535,21 @@ WeaponType GetWeaponType(int iEntity)
 	return WeaponType_Invalid;
 }
 
+TFClassType GetWeaponClassFilter(int iEntity)
+{
+	TFClassType nClass = TFClass_Unknown;
+	char sName[255];
+	GetEntPropString(iEntity, Prop_Data, "m_iName", sName, sizeof(sName));
+	for (int i = 0; i < sizeof(g_sClassNames); i++)
+	{
+		if (StrContains(sName, g_sClassNames[i], false) != -1)
+		{
+			nClass = view_as<TFClassType>(i);
```

Comment:
can just `return` here instead of needing `nClass` variable

Reply:
Tried but it wouldn't work